From foo@baz Tue Apr  9 12:12:43 2002
Date: Thu, 7 Sep 2006 15:12:05 -0700
From: Greg Kroah-Hartman <gregkh@suse.de>
To: Greg KH <greg@kroah.com>
Subject: [PATCH] We can not allow anonymous contributions to the kernel

The DCO does not mean anything if we allow anonymous contributors to the
kernel.  As this is an open source project, we need to do everything in
the open.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/SubmittingPatches |    2 ++
 1 file changed, 2 insertions(+)

--- gregkh-2.6.orig/Documentation/SubmittingPatches
+++ gregkh-2.6/Documentation/SubmittingPatches
@@ -308,6 +308,8 @@ then you just add a line saying
 
 	Signed-off-by: Random J Developer <random@developer.example.org>
 
+using your real name (sorry, no pseudonyms or anonymous contributions.)
+
 Some people also put extra tags at the end.  They'll just be ignored for
 now, but you can do this to mark internal company procedures or just
 point out some special detail about the sign-off. 
